featurebyte/query_graph/sql/interpreter/preview.py

```diff
@@ -469,6 +472,28 @@ def _is_dtype_supported(dtype: DBVarType, supported_dtypes: Optional[Set[DBVarTy
             return True
         return dtype in supported_dtypes
 
+    @staticmethod
+    def _get_cat_counts(
+        col_expr: expressions.Expression,
+        num_categories_limit: int = 500,
+    ) -> expressions.Select:
+        return (
+            expressions.select(
+                col_expr,
+                expressions.alias_(
+                    expressions.Count(this=make_literal_value("*")),
```

Comment:
nit: can consider standardizing the `count(*)` expression in a util to be reused across the code base.

we've a few different ways of doing it at the moment

- `expressions.Count(this=expressions.Star())`
- `expressions.Count(this="*")`
- the way you have it here

so might be nice to be consistent